src/main/webapp/js/grapheditor/Shapes.js

```diff
@@ -3687,6 +3746,25 @@ function getShape(cell) {
   mxUtils.extend(Metadata, mxArrowConnector);
   mxCellRenderer.registerShape('metadata', Metadata);
 
+  // Event
+  function Event() {
+    mxActor.call(this);
+  }
+  mxUtils.extend(Event, mxActor)
+  mxCellRenderer.registerShape('event', Event);
+
+  Event.prototype.redrawPath = function(c, x, y, w, h)
```

Comment:
Maybe calling it EventShape?

```suggestion
  function EventShape() {
    mxActor.call(this);
  }
  mxUtils.extend(EventShape, mxActor)
  mxCellRenderer.registerShape('event', EventShape);

  EventShape.prototype.redrawPath = function(c, x, y, w, h)
```

Reply:
Nah, forget it, half of what's here isn't called `[Something]Shape`.